src/tss2-esys/tss2-esys.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+?<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are you sure about this one ?

Reply:
Nope, I thought I got all those. When it doesn't know what to do, the tool throws a ? in there.